kubernetes/configurer/utils/ccd-import-definition.sh

```diff
@@ -12,3 +12,5 @@ curl -k --fail --show-error --silent -X POST \
   -H "Authorization: Bearer ${userToken}" \
   -H "ServiceAuthorization: Bearer ${serviceToken}" \
   -F file=@${1}
+
+echo ""
```

Comment:
How does it help?

Reply:
moves the cmd shell to a new line after the script returns making it more readable